package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "esvu",
-  "version": "1.2.13",
```

Comment:
can you remove this? version bump will be a separate commit

Reply:
Done